apps/web/cypress/tests/digest-playground.spec.ts

```diff
@@ -18,35 +19,41 @@ describe('Digest Playground Workflow Page', function () {
   });
 
   it('the set up digest workflow should redirec to template edit page', function () {
-    cy.intercept('**/notification-templates**').as('notificationTemplates');
-    cy.visit('/templates');
+    cy.intercept('GET', '**/notification-templates**').as('notificationTemplates');
+    cy.intercept('GET', '**/notification-templates/**').as('getNotificationTemplate');
```

Comment:
Isn't the first one including the second one?
I would expect the first one cover the second one, so I wonder if using just the first one all the time would cover all the interceptions we need.

Reply:
from what I saw in the debugger, no the first one is not including the second, they are separate